bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/airbase/AirbaseEnums.java

```diff
@@ -60,6 +60,7 @@ public static AirbaseMode fromValue(int value) {
 
     public enum AirbaseFanSpeed {
         // level,f_auto,f_airside
+        AUTO         (0, false, false),
```

Comment:
This class has a field called `auto`. Why isn't it set to true for this new enum value?

Reply:
Could you rephrase the question please?

Reply:
Are you referring to f_auto? f_auto applies to AUTO_LEVEL_N. When the Fan is set to "auto" (not auto_level_n), f_auto is set to false (0). That's just how the controller works.

Reply:
understood